### ISSUE TITLE
exportDefault incorrectly curries function

### ISSUE THREAD

**Issue by dudleycarr** [fable-compiler/Fable#2323]
### Description

Passing a function to `exportDefault` should directly expose the function. Instead, the exported function is wrapped in another a curried function.

### Repro code

```fsharp
let handler a b =
    [|a; b|]

exportDefault handler
```

```js
export function handler(a, b) {
    return [a, b];
}

export default ((a) => ((b) => handler(a, b)));
```

### Expected and actual results

```js
export function handler(a, b) {
    return [a, b];
}

export default handler;
```

### Related information

```shell
$ dotnet fable --version
Fable: F# to JS compiler 3.0.2
```

**Comment by dudleycarr** [comment]
@alfonsogarciacaro thanks!

**Comment by alfonsogarciacaro** [comment]
Thanks for reporting! Should be fixed in 3.0.4. If you have problems with uncurrying again, you can always make it explicit by converting the lambdas to delegates

```fsharp
// Using System.Func. Add an extra generic placeholder for the return time
let f = Func<_,_,_>(f x y -> x + y)

// Declaring a delegate
type MyFunc = delegate of int * int -> int
let f = MyFunc(f x y -> x + y)
```
